poe-tasks/get-modified-connectors.sh

```diff
@@ -125,9 +129,28 @@ print_list() {
 
 # Allow empty arrays without 'unbound variable' error from here on out.
 set +u
+# 10) If --local-cdk flag is set, also add Java connectors with useLocalCdk = true regardless of changes.
+if $LOCAL_CDK; then
+  echo "Finding Java Bulk CDK connectors with version = local..." >&2
+
+  for connector_dir in airbyte-integrations/connectors/*; do
+    if [[ -d "$connector_dir" ]]; then
+      # Check if it's a Java connector
+      if [ -f "$connector_dir/build.gradle" ]; then
+        connector_name=$(basename "$connector_dir")
+         # Search for cdk = 'local' or cdk = "local" in airbyteBulkConnector block
+         if grep -q "airbyteBulkConnector" "$connector_dir/build.gradle" && grep -q "cdk *= *['\"]local['\"]" "$connector_dir/build.gradle"; then
```

Comment:
This probably needs to conditionally check for `.kts` build scripts too.

Reply:
done